react/src/components/Widget/WidgetContainer.tsx

```diff
@@ -118,15 +122,15 @@ export const WidgetContainer: React.FC<WidgetContainerProps> = withSnackbar(
       (transaction: any, satoshis: number) => {
         if (sound && !hideToasts) txSound.play().catch(() => {});
 
-        const receivedAmount = satoshisToBch(satoshis);
+        const receivedAmount = satoshis;
 
         if (!hideToasts)
           // TODO: This assumes only bch
```

Comment:
Is this out of scope for adding XEC support?

Reply:
Removing `sats` and `bits` as currencies was a precursor to adding `xec`. Just fixing bugs related to that as i go since its within the same stacked branch